otterdog/eclipse-pass.jsonnet

```diff
@@ -17,7 +17,18 @@ orgs.newOrg('eclipse-pass') {
       default_workflow_permissions: "write",
     },
   },
+  credentials+: [{
```

Comment:
there is a misunderstanding, you probably looked at the readme at https://gitlab.eclipse.org/eclipsefdn/security/otterdog/#setup. This is mainly for setting up the tool to be able to manage different GitHub organizations. For the Eclipse Foundation this list is maintained here: https://github.com/EclipseFdn/otterdog-configs/

You should not be concerned by that.

Reply:
OK.  I will remove.